test/manipulation.test.js

```diff
@@ -57,6 +57,20 @@ describe('manipulation', function () {
       });
     });
 
+    it('should create instance (promise variant)', function (done) {
+      Person.create({name: 'Anatoliy'})
+      .then (function (p) {
```

Comment:
On the second thought, I think @ritch is right that the whitespace is not a deal breaker, especially since we don't have a linter set up yet. Let's focus on fixing the issues in the functionality to get this patch landed.